src/qt/forms/researcherwizardpoolpage.ui

```diff
@@ -91,7 +91,7 @@
    <item alignment="Qt::AlignVCenter">
     <widget class="QLabel" name="poolParagraphLabel">
      <property name="text">
-      <string>In pool mode, Gridcoin will not claim research rewards directly. The pool will collect and distribute research rewards on your behalf. Your wallet will earn staking rewards only (you do not need a BOINC account, CPID, or beacon). Please choose a pool and follow the instructions on the website to sign up and connect the pool's account manager to BOINC:</string>
+      <string>In this mode, a pool will take care of staking research rewards for you. Your wallet can still earn standard staking rewards on your balance. You do not need a BOINC account, CPID, or beacon. Please choose a pool and follow the instructions on the website to sign up and connect the pool's account manager to BOINC:</string>
```

Comment:
Does clicking or double clicking on a pool open it to the website? When people see the word choose, they may just try and click on the pool they want instead of knowing to go to the website listed there

Reply:
Yes.